webapi/src/main/java/org/sakaiproject/webapi/controllers/TasksController.java

```diff
@@ -127,17 +131,19 @@ public List<UserTaskAdapterBean> getSiteTasks(@PathVariable String siteId) throw
             }).collect(Collectors.toList());
     }
     
-    @GetMapping(value = "/tasks/role/{siteId}/{userId}", produces = MediaType.APPLICATION_JSON_VALUE)
-    public Map<String, String> getCurrentUserRoleOnSite(@PathVariable String siteId, @PathVariable String userId) {
+    @GetMapping(value = "/tasks/role/{siteId}/{userId}")
```

Comment:
this rest path makes no sense at all. Why not make it "/sites/{siteId}/users/current/isSiteUpdater"? That is self documenting, imo. On the other hand, if this is in a site, why not add some more permissions around tasks, rather than trying to guess roles?